docs/UniqueIDs.rst

```diff
@@ -0,0 +1,92 @@
+Unique IDs
+==========
+
+The database generates unique IDs and they are accessible from all nodes and relationships using the ``neo_id`` method. These keys are somewhat volatile and may be reused or change throughout a database's lifetime, so they are unsafe to use within an application.
+
+To work around this, you can define which key should act as primary key on ``Neo4j::ActiveNode`` classes instead of using the internal Neo4j ids. By default, ActiveNode will generate a unique ID using ``SecureRandom::uuid``. The instance methods ``id`` and ``uuid`` will both point to this.
```

Comment:
"To work around this, you can define" makes it sound like it's optional.  You mention "by default" below, but I think this should be clear outside of the gate that some `id_property` is required for every model, and if you don't define one, one will be defined for you.

Also, I don't think that `uuid` will point to the `id_property` when you define an `id_property` other than `uuid`, right?  I think it's just `id` / `id=`